browser.js

```diff
@@ -14,4 +14,4 @@ if (typeof WebSocket !== 'undefined') {
   ws = self.WebSocket || self.MozWebSocket
 }
 
-module.exports = ws
+export default ws
```

Comment:
It looks like a breaking change, the major version number should be changed when we publish it🤔